examples/genomicsdb_common.py

```diff
@@ -53,37 +53,49 @@ def parse_vidmap_json(vidmap_file, intervals=None):
     all_intervals = not intervals
     if not intervals:
         intervals = []
+    column_offset = 0
     for contig in contigs:
         if isinstance(contig, str):  # Old style vidmap json
             contig_name = contig
-            contigs_map[contig] = {
+            contig_elem = {
```

Comment:
We may not need this, but doesn't `contig_elem` also have a `name` in the `vcf2genomicsdb_init` case?

Reply:
Yes, it does in the `vcf2genomicsdb_init` case, but are really ignoring as the contigs map is a dict for faster lookup.